tools/rostopic/src/rostopic/__init__.py

```diff
@@ -1297,10 +1307,13 @@ def _rostopic_cmd_hz(argv):
     parser.add_option("--wall-time",
                       dest="use_wtime", default=False, action="store_true",
                       help="calculates rate using wall time which can be helpful when clock isnt published during simulation")
+    parser.add_option("--publish-hz",
+                    dest="publish_hz", default=False, action="store_true",
+                    help="publishes the rate on ~hz to be used by other program (e.g. for plotting)")
```

Comment:
Would it make sense to pass a topic name here? Otherwise if the user runs multiple of these commands it becomes difficult to choose the "right" one.